packages/jaeger-ui/src/components/DependencyGraph/index.test.js

```diff
@@ -78,21 +75,23 @@ describe('<DependencyGraph>', () => {
 
   describe('graph types', () => {
     it('renders a menu with options for the graph types', () => {
-      expect(wrapper.find(Tabs.TabPane).length).toBe(Object.keys(GRAPH_TYPES).length);
-      expect(wrapper.find({ tab: GRAPH_TYPES.FORCE_DIRECTED.name }).length).toBe(1);
-      expect(wrapper.find({ tab: GRAPH_TYPES.DAG.name }).length).toBe(1);
+      expect(wrapper.props().items.length).toBe(Object.keys(GRAPH_TYPES).length);
+      expect(wrapper.props().items[0].name).toBe(Object.keys(GRAPH_TYPES)[0].name);
+      expect(wrapper.props().items[1].name).toBe(Object.keys(GRAPH_TYPES)[1].name);
     });
 
     it('renders a force graph when FORCE_GRAPH is the selected type', () => {
       wrapper.simulate('change', GRAPH_TYPES.FORCE_DIRECTED.type);
       expect(wrapper.state('graphType')).toBe(GRAPH_TYPES.FORCE_DIRECTED.type);
-      expect(wrapper.find(DependencyForceGraph).length).toBe(1);
+      expect(wrapper.props().activeKey).toBe(GRAPH_TYPES.FORCE_DIRECTED.type);
+      expect(wrapper.props().activeKey).not.toBe(GRAPH_TYPES.DAG.type);
```

Comment:
this is an odd check. It should always pass if the previous passes, and it may also pass for many other unrelated arguments to `.toBe()`

Reply:
I thought it would add an extra line of surety. Though I agree with your statement. Removing 🚀